Pages/Index.cshtml.cs

```diff
@@ -38,57 +38,56 @@ private List<Brewery> GetBreweryData()
             }
             ViewData["Brand"] = brand;
 
-            var task = client.GetAsync("https://api.openbrewerydb.org/breweries");
-            HttpResponseMessage result = task.Result;
+            HttpResponseMessage result = await client.GetAsync("https://api.openbrewerydb.org/breweries");
             List<Brewery> breweries = new List<Brewery>();
 
             if (result.IsSuccessStatusCode)
             {
-                Task<string> readString = result.Content.ReadAsStringAsync();
-                string jsonString = readString.Result;
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("Data/breweryschema.json"));
-                JArray jsonarray = JArray.Parse(jsonString);
+                string breweryJson = await result.Content.ReadAsStringAsync();
+                JSchema schema = JSchema.Parse(await System.IO.File.ReadAllTextAsync("Data/breweryschema.json"));
+                JArray jsonarray = JArray.Parse(breweryJson);
                 IList<string> validationEvents = new List<string>();
 
                 if (jsonarray.IsValid(schema, out validationEvents))
                 {
-                    breweries = Brewery.FromJson(jsonString);
+                    breweries = Brewery.FromJson(breweryJson);
                 }
                 else
                 {
                     foreach (string evt in validationEvents)
                     {
-                        Console.WriteLine(evt);
+                        _logger.LogError(evt);
```

Comment:
Improved logging consistency, centralized logging, and better testability

Reply:
Reeeeeeeeeeally good idea.